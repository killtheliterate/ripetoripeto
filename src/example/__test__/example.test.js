import React from 'react'
import { shallow } from 'enzyme'

import Example from '../index'

describe('Example', () => {
  it('should render', () => {
    expect(shallow(<Example />)).toMatchSnapshot()
  })
})
