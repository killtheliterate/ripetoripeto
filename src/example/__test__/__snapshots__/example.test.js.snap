// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Example />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": "This is an example",
    },
    "ref": null,
    "rendered": "This is an example",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "This is an example",
      },
      "ref": null,
      "rendered": "This is an example",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
